packages/flutter/lib/src/widgets/transitions.dart

```diff
@@ -1124,6 +1124,50 @@ class AlignTransition extends AnimatedWidget {
 /// Animated version of a [DefaultTextStyle] that animates the different properties
 /// of its [TextStyle].
 ///
+/// {@tool dartpad --template=stateful_widget_material_ticker}
+///
+/// The following code implements the [DefaultTextStyleTransition] that shows a transition between thick blue font and thin red font.
```

Comment:
nit: can you add line breaks to the sentence to keep the number of characeters under 80 per line.

Reply:
@goderbauer Thank you for letting me know it. I fixed it.